toolkit/scripts/analysis.mk

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Observation: should we not add this as an include in the uber `Makefile` so that we can execute these targets :)
Something like :
`include $(SCRIPTS_DIR)/analysis.mk`

PS: maybe i am missing something trivial :(

Reply:
Good catch, forgot to checkout the main makefile as well :)